src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1169,8 +1180,48 @@ void uct_rc_mlx5_iface_common_event_cq(uct_ib_iface_t *ib_iface,
     iface->cq[dir].cq_sn++;
 }
 
+static UCS_F_ALWAYS_INLINE void
+uct_rc_mlx5_iface_emplace_compressed_cqes(uct_ib_mlx5_cq_t *cq,
+                                          unsigned cqe_title_index, int is_rx)
+{
+    uct_ib_mlx5_cq_decomp_t *cq_decomp = &cq->cq_decomp;
+    struct mlx5_cqe64 *title = uct_ib_mlx5_get_cqe(cq, cqe_title_index);
```

Comment:
align =

Reply:
Did I understand your proposal correctly?

```suggestion
    struct mlx5_cqe64 *title           = uct_ib_mlx5_get_cqe(cq,
                                                             cqe_title_index);
```